src/ESLint.Formatter/package.json

```diff
@@ -34,5 +34,8 @@
     "lodash": "^4.17.14",
     "mocha": "^7.1.2",
```

Comment:
Would you be OK with moving `chai` and `mocha` to `devDependencies` as well?

Reply:
Yes, thank you.

---
In reply to: [490201370](https://github.com/microsoft/sarif-sdk/pull/2071#discussion_r490201370) [](ancestors = 490201370)